### ISSUE TITLE
Exception on delete schedule 

### ISSUE THREAD

**Issue by vqmarkman** [sundeck-io/OpsCenter#450]
```
ile "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 561, in _run_script
    self._session_state.on_script_will_rerun(rerun_data.widget_states)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/streamlit/runtime/state/safe_session_state.py", line 68, in on_script_will_rerun
    self._state.on_script_will_rerun(latest_widget_states)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/streamlit/runtime/state/session_state.py", line 482, in on_script_will_rerun
    self._call_callbacks()
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/streamlit/runtime/state/session_state.py", line 495, in _call_callbacks
    self._new_widget_state.call_callback(wid)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/streamlit/runtime/state/session_state.py", line 247, in call_callback
    callback(*args, **kwargs)
File "/Users/vickymarkman/git-repo/OpsCenter/app/ui/warehouses.py", line 288, in <lambda>
    lambda row: self.on_delete_click(create_callback(data, row, weekday=True)),
File "/Users/vickymarkman/git-repo/OpsCenter/app/ui/base.py", line 132, in on_delete_click
    outcome = self.on_delete_click_internal(*args)
File "/Users/vickymarkman/git-repo/OpsCenter/app/ui/warehouses.py", line 236, in on_delete_click_internal
    [i.update(txn, i) for i in new_warehouses]
File "/Users/vickymarkman/git-repo/OpsCenter/app/ui/warehouses.py", line 236, in <listcomp>
    [i.update(txn, i) for i in new_warehouses]
File "/Users/vickymarkman/git-repo/OpsCenter/app/ui/crud.zip/crud/base.py", line 83, in update
    session.sql(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/telemetry.py", line 139, in wrap
    result = func(*args, **kwargs)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/dataframe.py", line 586, in collect
    return self._internal_collect_with_tag_no_telemetry(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/dataframe.py", line 633, in _internal_collect_with_tag_no_telemetry
    return self._session._conn.execute(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 451, in execute
    result_set, result_meta = self.get_result_set(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/analyzer/snowflake_plan.py", line 181, in wrap
    raise ne.with_traceback(tb) from None
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/analyzer/snowflake_plan.py", line 111, in wrap
    return func(*args, **kwargs)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 552, in get_result_set
    result = self.run_query(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 102, in wrap
    raise ex
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 96, in wrap
    return func(*args, **kwargs)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 366, in run_query
    raise ex
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/snowpark/_internal/server_connection.py", line 347, in run_query
    results_cursor = self._cursor.execute(query, params=params, **kwargs)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/cursor.py", line 788, in execute
    kwargs["binding_params"] = self._connection._process_params_qmarks(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/connection.py", line 1359, in _process_params_qmarks
    snowflake_type, snowflake_binding = get_type_and_binding(v)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/connection.py", line 1317, in _get_snowflake_type_and_binding
    Error.errorhandler_wrapper(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/errors.py", line 290, in errorhandler_wrapper
    handed_over = Error.hand_to_other_handler(
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/errors.py", line 345, in hand_to_other_handler
    cursor.errorhandler(connection, cursor, error_class, error_value)
File "/Users/vickymarkman/Library/Caches/pypoetry/virtualenvs/opscenter-g7Dx2M1v-py3.8/lib/python3.8/site-packages/snowflake/connector/errors.py", line 221, in default_errorhandler
    raise error_class(
```

**Comment by joshelser** [comment]
My guess is that I caused this in #432 -- I remember a similar bug in dealing with the `pandas.NaT`

**Comment by joshelser** [comment]
Will be fixed with the revert in #451. Sorry about that! Need to figure out what tests need to be added here.
